ansible/roles/test/tasks/fast-reboot.yml

```diff
@@ -0,0 +1,95 @@
+- block:
+    - fail: msg="Please set ptf_host variable"
+      when: ptf_host is not defined
+
+    - name: Remove existing ip from ptf host
+      script: roles/test/files/helpers/remove_ip.sh
+      delegate_to: "{{ ptf_host }}"
+
+    - name: Make all mac addresses in ptf unique - should be done in vm_set
```

Comment:
so currently this is here. will there be any plans to move this part to the testbed setup procedure? I think for the t0 scenario, it is required to have all the MAC different on all the VLAN neighbors and then IP could be assigned to each of the interfaces.